addons/website_slides/models/slide_channel.py

```diff
@@ -26,6 +26,18 @@ class ChannelUsersRelation(models.Model):
     completed_slides_count = fields.Integer('# Completed Slides')
     partner_id = fields.Many2one('res.partner', index=True, required=True, ondelete='cascade')
     partner_email = fields.Char(related='partner_id.email', readonly=True)
+    user_id = fields.Many2one('res.users', string='Responsible', related='channel_id.user_id')
+    channel_type = fields.Selection(related='channel_id.channel_type')
+    visibility = fields.Selection(related='channel_id.visibility')
+    enroll = fields.Selection(related='channel_id.enroll')
+    website_id = fields.Many2one('website', string='Website', related='channel_id.website_id')
+
+    def name_get(self):
+        result = []
+        for rec in self:
+            name = rec.channel_id.display_name + ' (%s)' % rec.partner_id.display_name
+            result.append((rec.id, name))
+        return result
```

Comment:
I don't see anything about this part in the task specifications.
Can you explain the changes?